codecov.yml

```diff
@@ -6,3 +6,7 @@ coverage:
     patch:
       default:
         informational: true
+
+comment:
+  layout: "reach, diff, flags, files"
+  behavior: "off"
```

Comment:
This yaml doesn't validate. 

```
➜  codecov_yaml curl --data-binary @codecov.yml https://codecov.io/validate

Error at ['comment', 'behavior']: 
must be of string type
```

You want:

```
comment: false
```
Docs: https://docs.codecov.com/docs/pull-request-comments#disable-comment